e2e/protractor/suites/actions/library/library-actions.test.ts

```diff
@@ -1,429 +0,0 @@
-
-      expect(await dataTable.getLibraryRole(siteSearchModerated1Admin)).toEqual(SITE_ROLES.NONE.LABEL);
-      const hasJoinRequest = await apis.user.sites.hasMembershipRequest(siteSearchModerated1Admin);
-      expect(hasJoinRequest).toBe(true, `Join request does not exist on ${siteSearchModerated1Admin}`);
-    });
-  });
-
-  describe('Leave a library', () => {
-    const adminSites = [sitePublic2Admin, sitePublic3Admin, sitePublic4Admin, sitePublic5Admin];
-    const userSites = [sitePublicUser];
-
-    beforeAll(async () => {
-      await adminApiActions.sites.createSites(adminSites);
-      await apis.user.sites.createSites(userSites);
-
-      await apis.user.favorites.addFavoriteById('site', sitePublic3Admin);
-      await adminApiActions.sites.addSiteMember(sitePublic2Admin, username, SITE_ROLES.SITE_COLLABORATOR.ROLE);
-      await adminApiActions.sites.addSiteMember(sitePublic3Admin, username, SITE_ROLES.SITE_MANAGER.ROLE);
-      await adminApiActions.sites.addSiteMember(siteSearchPublic2Admin, username, SITE_ROLES.SITE_CONTRIBUTOR.ROLE);
-      await adminApiActions.sites.addSiteMember(sitePublic4Admin, username, SITE_ROLES.SITE_MANAGER.ROLE);
-      await adminApiActions.sites.addSiteMember(sitePublic5Admin, username, SITE_ROLES.SITE_MANAGER.ROLE);
-      await apis.user.queries.waitForSites(siteSearchPublic2Admin, { expect: 1 });
-    });
-
-    afterAll(async () => {
-      await adminApiActions.deleteSites(adminSites);
-      await apis.user.sites.deleteSites(userSites);
-    });
-
-    it('[C290106] from My Libraries', async () => {
-      await page.goToMyLibrariesAndWait();
-      await dataTable.selectItem(sitePublic2Admin);
-      await BrowserActions.click(toolbar.leaveButton);
-      await page.waitForDialog();
-      await BrowserActions.click(confirmDialog.okButton);
-
-      expect(await page.getSnackBarMessage()).toEqual(`You have left the library`);
-      expect(await dataTable.isItemPresent(sitePublic2Admin)).toBe(false, `${sitePublic2Admin} is displayed`);
-    });
-
-    it('[C290110] from Favorite Libraries', async () => {
-      await page.goToFavoriteLibrariesAndWait();
-      await dataTable.selectItem(sitePublic3Admin);
-      await BrowserActions.click(toolbar.leaveButton);
-      await page.waitForDialog();
-      await BrowserActions.click(confirmDialog.okButton);
-
-      expect(await page.getSnackBarMessage()).toEqual(`You have left the library`);
-      expect(await dataTable.isItemPresent(sitePublic3Admin)).toBe(true, `${sitePublic3Admin} is not displayed`);
-    });
-
-    it('[C306961] from Search Results', async () => {
-      await toolbar.clickSearchIconButton();
-      await searchInput.clickSearchButton();
-      await searchInput.checkLibraries();
-      await searchInput.searchForLibrary(siteSearchPublic2Admin);
-      await dataTable.waitForBody();
-
-      await dataTable.selectItem(siteSearchPublic2Admin);
-      await BrowserActions.click(toolbar.leaveButton);
```

Comment:
leaveButton is unused now - can be removed

Reply:
removed in [aa865dc](https://github.com/Alfresco/alfresco-content-app/pull/3545/commits/aa865dcf243348da373c4560117ab99260fb3fe8)